<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>navdialog</class>
 <widget class="QDialog" name="navdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>820</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>820</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1250</width>
    <height>820</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>导航</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>490</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>生成路径：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>490</y>
     <width>221</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="way_comboBox">
      <item>
       <property name="text">
        <string>选择出行方式</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最短路径</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最短步行时间</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最短骑行时间</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="startnode_comboBox">
      <item>
       <property name="text">
        <string>选择起点</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="endnode_comboBox">
      <item>
       <property name="text">
        <string>选择终点</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="start_pushButton">
      <property name="text">
       <string>生成导航路线</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="shahe">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>0</y>
     <width>621</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>shahemap.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="benbu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>641</width>
     <height>841</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>buptmap201608.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
